File: src/main/java/org/taktik/icure/services/external/rest/v1/dto/embed/FlatRateTarificationDto.java
package org.taktik.icure.services.external.rest.v1.dto.embed;

import java.io.Serializable;
import java.util.Map;
import java.util.Set;

public class FlatRateTarificationDto implements Serializable {

    private static final long serialVersionUID = 1L;

    protected String code;
    protected java.util.Map<String, String> label;
    protected Set<ValorisationDto> valorisations;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public Map<String, String> getLabel() {
        return label;
    }

    public void setLabel(Map<String, String> label) {
        this.label = label;
    }

    public Set<ValorisationDto> getValorisations() {
        return valorisations;
    }

    public void setValorisations(Set<ValorisationDto> valorisations) {
        this.valorisations = valorisations;
    }
}

File: src/main/kotlin/org/taktik/icure/services/external/rest/v1/dto/embed/FlatRateTarificationDto.kt
package org.taktik.icure.services.external.rest.v1.dto.embed

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude
import com.github.pozo.KotlinBuilder
import java.io.Serializable

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
@KotlinBuilder
data class FlatRateTarificationDto(
        val code: String? = null,
        val flatRateType: FlatRateTypeDto? = null,
        val label: Map<String, String>? = null,
        val valorisations: List<ValorisationDto> = listOf(),
        override val encryptedSelf: String? = null
) : EncryptedDto, Serializable
